spec/integration/init_web_spec.cr

```diff
@@ -10,6 +10,20 @@ describe "Initializing a new web project" do
       should_run_successfully "crystal build src/test_project.cr"
       should_run_successfully "crystal src/app.cr"
     end
+    FileUtils.rm_r "test-project"
```

Comment:
I think this should go in a begin/ensure so that it's always deleted at the end. Something like this:

```crystal
begin do
  # the test code
ensure
  FIleUtils.rm_r "test-project"
end
```

Reply:
Yes, that would be the best way to do it. I will update the code.